dist/src/main/java/io/camunda/identity/controllers/IdentityIndexController.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.identity.controllers;
+
+import io.camunda.webapps.controllers.WebappsRequestForwardManager;
+import jakarta.servlet.ServletContext;
+import jakarta.servlet.http.HttpServletRequest;
+import java.io.IOException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+@Controller
+public class IdentityIndexController {
+
+  @Autowired private ServletContext context;
+
+  @Autowired private WebappsRequestForwardManager webappsRequestForwardManager;
```

Comment:
Would it make sense to move to constructor-based injection? I've always got that in the back of my head for Spring (testability, immutability, better null-safety, etc.)... Wdyt? 😃

Reply:
I like the suggestion, and ordinarily I would go for constructor injection however this time in an effort to keep pace I would prefer to address it later on, this also aligns with the other classes providing the same functionality so it could be a common change :)

Reply:
Totally makes sense :)